example/provision-aks-via-resource/variables.tf

```diff
@@ -0,0 +1,130 @@
+# Infra variables
+
+variable "subscription_id" {
+  description = "The Azure subscription ID is a unique identifier that represents your subscription to Microsoft Azure."
+  type        = string
+}
+
+variable "namespace" {
+  description = "The namespace is a user-defined name that provides a unique identifier for organizing and grouping your resources within Azure."
+  type        = string
+}
+
+variable "location" {
+  description = "The location refers to the physical region or datacenter where your Azure resources are deployed."
+  type        = string
+}
+
+variable "create_resource_group" {
+  description = "The 'create_resource_group' variable is a boolean flag that determines whether to create a resource group (RG) in Azure."
+  type        = bool
+  default     = true
+}
+
+# launch-helm variables
+variable "helm_repository" {
+  description = "Helm repository URL"
+  type        = string
+  default     = "https://wandb.github.io/helm-charts"
```

Comment:
charts.wandb.ai

Reply:
Hey Justin, could you elaborate on that? 

Currently charts.wandb.ai is quite bare. Are you planning to add more info to help segregate W&B server, W&B Operator, and W&B Launch(I can help with that one).